code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -231,19 +231,65 @@
 	max_combined_w_class = 27
 	slowdown = 1
 
-/obj/item/weapon/storage/backpack/duffel/syndimed
+/obj/item/weapon/storage/backpack/duffel/syndie
+	name = "suspicious looking dufflebag"
+	desc = "A large dufflebag for holding extra tactical supplies."
+	icon_state = "duffel-syndi"
+	item_state = "duffel-syndimed"
+	origin_tech = "syndicate=1"
+	silent = 1
+	slowdown = 0
+
+/obj/item/weapon/storage/backpack/duffel/syndie/med
 	name = "suspicious duffelbag"
 	desc = "A black and red duffelbag with a red and white cross sewn onto it."
 	icon_state = "duffel-syndimed"
 	item_state = "duffel-syndimed"
-	slowdown = 0
 
-/obj/item/weapon/storage/backpack/duffel/syndiammo
+/obj/item/weapon/storage/backpack/duffel/syndie/ammo
 	name = "suspicious duffelbag"
 	desc = "A black and red duffelbag with a patch depicting shotgun shells sewn onto it."
 	icon_state = "duffel-syndiammo"
 	item_state = "duffel-syndiammo"
-	slowdown = 0
+
+/obj/item/weapon/storage/backpack/duffel/syndie/ammo/loaded
+	desc = "A large dufflebag, packed to the brim with Bulldog shotgun ammo."
+
+	New()
+		..()
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g(src)
+		new /obj/item/ammo_box/magazine/m12g/buckshot(src)
+		new /obj/item/ammo_box/magazine/m12g/stun(src)
+		new /obj/item/ammo_box/magazine/m12g/dragon(src)
+		return
+
+/obj/item/weapon/storage/backpack/duffel/syndie/surgery
+	name = "surgery dufflebag"
+	desc = "A suspicious looking dufflebag for holding surgery tools."
+	icon_state = "duffel-syndimed"
+	item_state = "duffle-syndimed"
+	storage_slots = 12
+
+	New()

```

Comment:
Didn't look at it close enough; I had just worked with syringe code (which is relatively pathed)--I didn't look close enough and thought this was too.